pkg/nsx/services/vpc/vpc.go

```diff
@@ -714,7 +752,7 @@ func (service *VPCService) buildAVIGroupTag(vpcId string) []model.Tag {
 			Tag:   common.String(strings.Join(common.TagValueVersion, ".")),
 		},
 		{
-			Scope: common.String(common.TagScopeVPCCRUID),
+			Scope: common.String(common.TagScopeNamespaceUID), // vpcId is ns uid
```

Comment:
This scope changed from `TagScopeVPCCRUID` to `TagScopeNamespaceUID`, is there any related change? i.e. where consumes this tag?

Reply:
I think we can remove this tag, @TaoZou1 pls help confirm.

Reply:
yes

Reply:
OK，removed